recipes/libgeotiff/all/conanfile.py

```diff
@@ -115,7 +115,7 @@ def package_info(self):
         self.cpp_info.set_property("cmake_module_file_name", "GeoTIFF")
         self.cpp_info.set_property("cmake_build_modules", [self._module_vars_file])
         self.cpp_info.set_property("cmake_file_name", "geotiff")
-        self.cpp_info.set_property("cmake_target_name", "geotiff_library")
+        self.cpp_info.set_property("cmake_target_name", "geotiff::geotiff")
```

Comment:
The upstream project generate `geotiff_library` as it was already in the recipe - our goal is to reflect the usage to preserve compatibility with what upstream generates.